kubetest/aksengine.go

```diff
@@ -1224,10 +1224,10 @@ func (c *aksEngineDeployer) TestSetup() error {
 	} else if *testAzureFileCSIDriver || *testAzureDiskCSIDriver || *testBlobCSIDriver || *testSMBCSIDriver || *testNFSCSIDriver {
 		// Set env vars required by CSI driver e2e jobs.
 		// tenantId, subscriptionId, aadClientId, and aadClientSecret will be obtained from AZURE_CREDENTIAL
-		if err := os.Setenv("RESOURCE_GROUP", c.resourceGroup); err != nil {
+		if err := os.Setenv("AZURE_RESOURCE_GROUP", c.resourceGroup); err != nil {
```

Comment:
to ensure tests are not broken, we need split the changes into two parts: 1) kubetest changes and 2) manifest updates with test image bump.

Reply:
we should set both `RESOURCE_GROUP` and `AZURE_RESOURCE_GROUP` env vars in the beginning, otherwise it would be broken easily.

Reply:
test image update costs a few days sometimes

Reply:
can we use `RESOURCE_GROUP` since add `AZURE_` requires too many changes, and it does not make much difference?

Reply:
Added RESOURCE_GROUP and LOCATION back so we can merge this async